README.md

```diff
@@ -23,6 +23,33 @@ By participating, you are expected to uphold this code. Please report unacceptab
 
 PowSyBl Dev Tools are components meant solely for debugging various PowSyBl components - in particular for [PowSyBl Single Line Diagram](https://github.com/powsybl/powsybl-single-line-diagram).
 
+## PowSyBl diagram-viewer Dev Tool
+PowSyBl diagram-viewer Dev Tool is a JavaFX basic viewer used to display:
+* the single line diagrams generated by [PowSyBl Single Line Diagram](https://github.com/powsybl/powsybl-single-line-diagram)
+* the area diagrams generated by [PowSyBl Network Area Diagram](https://github.com/powsybl/powsybl-network-area-diagram).
+
+### Launching the viewer
+- First install 
+  - [powsybl-single-line-diagram](https://github.com/powsybl/powsybl-single-line-diagram) latest SNAPSHOT version (clone the repo then `mvn clean install` on `main` branch)
+  - [powsybl-network-area-diagram](https://github.com/powsybl/powsybl-network-area-diagram) latest SNAPSHOT version (clone the repo then `mvn clean install` on `main` branch)
```

Comment:
Those are the old repositories. You should refer to [powsybl-diagram](https://github.com/powsybl/powsybl-diagram) instead.